contrib/src/main/java/com/datatorrent/contrib/kafka/AbstractKafkaInputOperator.java

```diff
@@ -203,6 +204,14 @@ public AbstractKafkaInputOperator()
    */
   protected abstract void emitTuple(Message message);
```

Comment:
I haven't changed the method signature, that would create backward compatibility problems. I have introduced one more new method, and the default implementation of that method will call the existing emitTuple, so no change for existing users at all. The new method will help others to override the behavior if they need to.